kotest-assertions/kotest-assertions-json/src/jvmMain/kotlin/io/kotest/assertions/json/file/matchers.kt

```diff
@@ -0,0 +1,41 @@
+package io.kotest.assertions.json.file
+
+import io.kotest.assertions.json.CompareJsonOptions
+import io.kotest.assertions.json.shouldEqualJson
+import io.kotest.assertions.json.shouldNotEqualJson
+import io.kotest.assertions.json.shouldEqualSpecifiedJson
+import io.kotest.assertions.json.shouldEqualSpecifiedJsonIgnoringOrder
+import io.kotest.assertions.json.shouldNotEqualSpecifiedJson
+import java.io.File
+
+infix fun File.shouldEqualJson(expected: String): File {
+   this.readText().shouldEqualJson(expected)
+   return this
+}
```

Comment:
nit:

This can all be code-golfed to `infix fun File.shouldEqualJson(expected: String) = readText().shouldEqualJson(expected)`

Reply:
that prevents it from returning the instance of `File`. some matchers return the file, which is why I used this pattern, whereas other matchers don't, where I used that pattern.

though also, just realized that I forgot to add the validation for the `File` existing & being a `File` for this file.